src/config/konnectors.json

```diff
@@ -1404,7 +1404,7 @@
       "connector": true
     },
     "source": "git://gitlab.cozycloud.cc/gjacquart/cozy-konnector-orangemobile.git#build",
-    "timeInterval": [15, 21],
+    "frequency": "hourly",
```

Comment:
According to [here](https://github.com/cozy/cozy-collect/blob/b8fb26ef9cae8fe47effbca83f33f5f43cfe1346/src/ducks/triggers/index.js#L8), the only two valid values are `daily` or `weekly` so I think the best that we can do for now is `daily`.